frame/balances/src/lib.rs

```diff
@@ -456,22 +458,35 @@ decl_module! {
 		) {
 			ensure_root(origin)?;
 			let who = T::Lookup::lookup(who)?;
+			let existential_deposit = T::ExistentialDeposit::get();
+			let mut final_free = new_free;
```

Comment:
more cleanly written as `let new_free = if new_free < existential_deposit { Zero::zero() } else { new_free };` since this doesn't leave an intermediate value around nor needs use of `mut`.